server/views/pages/functionalSkills/index.njk

```diff
@@ -0,0 +1,75 @@
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Home" %}
+
+{% set pageId = "all-functional-skills" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: dpsUrl
+      },
+      {
+        text: "Manage learning and work progress",
+        href: "/"
+      },
+      {
+        text: "Manage " + prisonerSummary.firstName | capitalize + " " + prisonerSummary.lastName | capitalize + "'s learning plan",
+        href: "/plan/" + prisonerSummary.prisonNumber + "/view/education-and-training"
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+  {% include "../../partials/prisonerBanner.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">Learning and work progress</span>
+      <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+
+      <table class="govuk-table" id="functional-skills-table">
+        <caption class="govuk-table__caption govuk-table__caption--s">Current functional skills</caption>
+
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Subject</th>
+          <th scope="col" class="govuk-table__header">Assessed on</th>
+          <th scope="col" class="govuk-table__header">Level</th>
+          <th scope="col" class="govuk-table__header">Assessment type</th>
+          <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Action</span></th>
```

Comment:
Is this the right way to do this @chrimesdev ?
The design basically has a 4th column, where there will be a Change link (not in this story), or the word "Curious" if the Functional Skill record came from Curious (we cannot change data in Curious, hence its not a Change link)

So what I've done here is add it as a 4th column in the table; here the column heading is hidden, but has the text "Action" to hopefully give screen readers a clue?

Is that the right approach / technique? Would you use some other word/phrase other than simply "Action" ?